special-interest-groups/infrastructure/meetings/2023-01-x.md

```diff
@@ -0,0 +1,47 @@
+# Infrastructure SIG Meeting 2023-01-x
```

Comment:
Can you please update the infrstructure/README.md with the generalized version of the meeting information? E.g. "Meetings shall be 30 min. Meetings shall be held 1 per quarter"

Reply:
Done.